services/batch.rst

```diff
@@ -36,15 +37,15 @@ Access to the **HTCondor Prod Cluster** is restricted to users working formal da
 
 Jobs can be submitted to the **Slurm Cluster** from any of the above submit nodes.
 
-To report system issues, log into `LSST JIRA <https://jira.lsstcorp.org/>`_ and file a `JIRA ticket in the IT Helpdesk Support <https://ls.st/ihsticket>`_ project tagging NCSA as the responsible organization.
+To report system issues, please submit an :doc:`IHS ticket <ldf-tickets>` tagging NCSA as the responsible organization.
 
 
 .. _batch-connect:
 
 Rubin Batch Systems: Connecting and Authenticating
 ==================================================
 
-The batch submit nodes can be accessed after first connecting to the :doc:`lsst-login <lsst-login>` nodes. Once connected to an ``lsst-login`` node a user can connect to a submit node via its short hostname (e.g., ``lsst-condordac-sub01``) without having to enter a password (Kerberos authentication should be used by default; if your Kerberos ticket expires on the login node you may need to ``kinit`` again before proceeding to the submit node).
+The batch submit nodes can be accessed after first connecting to the :doc:`lsst-login <lsst-login>` nodes. Once connected to an ``lsst-login`` node a user can connect to a submit node via its short hostname (e.g., ``lsst-devl01``) without having to enter a password (Kerberos authentication should be used by default; if your Kerberos ticket expires on the login node you may need to ``kinit`` again before proceeding to the submit node).
```

Comment:
@jakerundall Why does this say the following?

> via its short hostname (e.g., ``lsst-devl01``)

It works regardless of short or full hostname, correct?